hack/generate-values.sh

```diff
@@ -2,13 +2,15 @@
 
 set -eu
 
+: "${DOMAIN:?}"
+
 ## Usage: ./generate-values.sh <my-domain>
 ## where my-domain will be used as both the system domain and app domain.
 
 # Make sure bosh binary exists
 bosh --version >/dev/null
 
-VARS_FILE=/tmp/cf-vars.yaml
+VARS_FILE="/tmp/${DOMAIN}/cf-vars.yaml"
```

Comment:
We'll have to `mkdir -p /tmp/${DOMAIN}` sometime before this line.

Reply:
@jenspinney moved it to the next line, so the domain is available.